libtextsecure/outgoing_message.js

```diff
@@ -137,7 +137,17 @@ OutgoingMessage.prototype = {
 
         return Promise.all(deviceIds.map(function(deviceId) {
             var address = new libsignal.SignalProtocolAddress(number, deviceId);
-            var sessionCipher = new libsignal.SessionCipher(textsecure.storage.protocol, address);
+
+            var ourNumber = textsecure.storage.user.getNumber();
+            var theirNumber = address.toString().split('.')[0];
```

Comment:
Given that it is generated from the plain 'address', I suppose. My attempt at extreme clarity made things harder to read?? Whaa? :0)